src/HotChocolate/Fusion-vnext/src/Fusion.Composition/PreMergeValidation/Rules/LookupMustNotReturnListRule.cs

```diff
@@ -0,0 +1,38 @@
+using HotChocolate.Fusion.Events;
+using HotChocolate.Skimmed;
+using static HotChocolate.Fusion.Logging.LogEntryHelper;
+
+namespace HotChocolate.Fusion.PreMergeValidation.Rules;
+
+/// <summary>
+/// Fields annotated with the <c>@lookup</c> directive are intended to retrieve a single entity
+/// based on provided arguments. To avoid ambiguity in entity resolution, such fields must return
+/// a <b>single object</b> and not a <c>list</c>. This validation rule enforces that any field
```

Comment:
As mentioned, let's align the formatting with the spec, generally-speaking (use bold where the spec uses bold, etc.).